docs/concepts/privacy.md

```diff
@@ -0,0 +1,71 @@
+
+## Enhancing your IPFS privacy
+
+If there are situations in which you know you'll need to remain private but still want to use IPFS, one of the approaches outlined below may help. And don't forget, you can always discuss privacy and get others' input or ideas in the official [IPFS forums](https://discuss.ipfs.io).
+
+### Disabling reproviding
+
+By default, an IPFS node announces itself as a provider for every CID in its cache. If you'd like to disable this behavior, you can do so in the [reprovider settings](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#reprovider) of your node's config file.
+
+Disabling default reprovision still enables you to provide data that you've pinned, so it's still possible for you to provide other nodes with content that you care about and want to make sure continues to be available over IPFS.
+
+### Using a public gateway
+
+Using a public [IPFS gateway](/how-to/address-ipfs-on-web/#http-gateways) is one way to request IPFS-hosted content without revealing any information about your local node — because you aren't using a local node! However, this method does keep you from enjoying all the benefits of being a full participant in the IPFS network.
+
+Public IPFS gateways are primarily intended as a "bridge" between the legacy web and the distributed web; they allow ordinary web clients to request IPFS-hosted content via HTTP. That's great for back-compatibility, but if you only request content through public gateways rather than directly over IPFS, you're not actually part of the IPFS network; that gateway is the network participant, acting on your behalf. It's also important to remember that gateway operators could be collecting their own private metrics, which could include tracking the IP addresses that use a gateway and correlating those with what CIDs are requested.
+
+### Using Tor
+
+If you're familiar with [Tor](https://www.torproject.org/) and comfortable with the command line, you may wish to try [running IPFS over Tor transport](https://dweb-primer.ipfs.io/avenues-for-access/tor-transport) by configuring your node's settings.
+
+If you're a developer building on IPFS, it's worth noting that the global IPFS community continues to experiment with using Tor transport — see [this example from e-commerce organization OpenBazaar](https://github.com/OpenBazaar/go-onion-transport) — and there may already be an open-source codebase to help your own project achieve this.
+
+### Encrypting content transported via IPFS
+
+If your privacy concerns are less about the potential for monitoring and more about the visibility of the IPFS-provided content itself, this can be mitigated simply by encrypting the content before adding it to the IPFS network. While traffic involving the encrypted content could still be tracked, the _data_ represented by encrypted content's CIDs remains unreadable by anyone without the ability to decrypt it.
```

Comment:
Need to caveat the safety in encrypting content and putting it on a public network. This is a non-starter for some folks, as future breakthroughs in computing may allow going back and decrypting older public content.

Reply:
Good point: added in https://github.com/ipfs/ipfs-docs/pull/554/commits/429c780e82193a4511c86ef9b933f9da68473e43